plugins/woocommerce/src/Internal/Admin/Suggestions/PaymentExtensions/PaymentExtensionsData.php

```diff
@@ -0,0 +1,1440 @@
+		),
+		'LS' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'LR' => array(),
+		'LY' => array(),
+		'MG' => array(),
+		'MW' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'ML' => array(),
+		'MR' => array(),
+		'MU' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'MA' => array(
+			self::PAYONEER,
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'MZ' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'NA' => array(),
+		'NE' => array(),
+		'NG' => array(
+			self::PAYSTACK,
+		),
+		'RE' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'RW' => array(),
+		'ST' => array(),
+		'SN' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'SC' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'SL' => array(),
+		'SO' => array(),
+		'ZA' => array(
+			self::PAYFAST,
+			self::PAYPAL_FULL_STACK,
+			self::PAYSTACK,
+			self::PAYPAL_WALLET,
+		),
+		'SS' => array(),
+		'TZ' => array(),
+		'TG' => array(),
+		'TN' => array(),
+		'UG' => array(),
+		'EH' => array(),
+		'ZM' => array(),
+		'ZW' => array(),
+
+		// Middle East.
+		'BH' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'IQ' => array(),
+		'IL' => array(
+			self::AIRWALLEX,
+		),
+		'JO' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'KW' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'LB' => array(),
+		'OM' => array(
+			self::PAYMOB,
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'PK' => array(
+			self::PAYONEER,
+			self::PAYMOB,
+		),
+		'QA' => array(
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'SA' => array(
+			self::PAYMOB,
+			self::PAYPAL_FULL_STACK,
+			self::PAYPAL_WALLET,
+		),
+		'AE' => array(
+			self::WOOPAYMENTS,
+			self::PAYPAL_FULL_STACK,
+			self::STRIPE,
+			self::PAYONEER,
+			self::PAYMOB,
+			self::PAYPAL_WALLET,
+		),
+		'YE' => array(),
+	);
+
+	/**
+	 * Get the entire list of payment extensions details.
+	 *
+	 * @return array The list of all payment extensions details.
+	 */
+	private static function get_list(): array {
+		return array(
+			self::AIRWALLEX         => array(
+				'type'        => self::TYPE_PSP,
+				'title'       => esc_html__( 'Airwallex Payments', 'woocommerce' ),
+				'description' => esc_html__( 'Boost international sales and save on FX fees. Accept 60+ local payment methods including Apple Pay and Google Pay.', 'woocommerce' ),
+				'image'       => plugins_url( 'assets/images/onboarding/airwallex.png', WC_PLUGIN_FILE ),
+				'image_72x72' => plugins_url( 'assets/images/payment_methods/72x72/airwallex.png', WC_PLUGIN_FILE ),
+				'source'      => array(
+					'type' => self::SOURCE_TYPE_WPORG,
+					'slug' => 'airwallex-online-payments-gateway',
+				),
+				'links'       => array(
+					array(
+						'type' => self::LINK_TYPE_ABOUT,
```

Comment:
idea: add the link text too, so we don't need to map it on the client

Reply:
No no. That is a presentational thing and it should not be a concern for the suggestions data. At most, it would be a Payments Settings thing (a higher-up layer), but I am inclined to hard-code the menu item label in the Payments settings page JS since it should be consistent.

Reply:
okay, good point - type is fine, we can do it on higher-up level.